src/JabbR-Core/userData.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace JabbR_Core
+{
+    // Class object to replace repo and EF models.. hard coded in Chat.cs for invoke functions
+    // user data is gathered upon login/auth which populates models
+
+    public class UserData
```

Comment:
Heather, just add { get; set; } to each at the end of each line ex public string Name { get; set; }